docs/python/testing.md

```diff
@@ -92,7 +92,7 @@ With this code, you can experience working with tests in VS Code as described in
 
 Testing in Python is disabled by default. To enable testing, use the **Python: Configure Tests** command on the Command Palette. This command prompts you to select a test framework, the folder containing tests, and the pattern used to identify test files.
 
-You can also configure testing manually by setting *one and only one* of the following settings to true: `python.testing.unittestEnabled`, `python.testing.pytestEnabled`, and `python.testing.nosetestsEnabled`. Each framework also has specific configuration settings as described under [Test configuration settings](#test-configuration-settings) for their folders and patterns.
+You can also configure testing manually by setting *one and only one* of the following settings to true: `python.testing.unittestEnabled` or `python.testing.pytestEnabled`. Each framework also has specific configuration settings as described under [Test configuration settings](#test-configuration-settings) for their folders and patterns.
```

Comment:
Do we need to mention that this is per workspace?

Reply:
but can't they configure it both on user and workspace settings?